src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.spec.ts

```diff
@@ -1186,4 +1191,294 @@ describe('HearingEditSummaryComponent', () => {
     const expected = cold('(b|)', { b: toShow });
     expect(result$).toBeObservable(expected);
   });
+
+  // New tests for change to
```

Comment:
Incomplete comment?

Reply:
Should have removed this, it was for me to identify the new test I was creating.  This has been removed.